### PR TITLE
Card #6

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
Card : #4, #5, #6

https://trello.com/c/FmknUKmQ
https://trello.com/c/qqpvgMv6
https://trello.com/c/ZFJl97DC

Implemented Readmore post feature.

### CODE REVIEW

**Comment by aloka-cuelogic** [comment]
@sayalis-cuelogic 

Please resolve conflicts and re-raise the PR

File: commentpost/templates/commentpost/comments.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{% load staticfiles %}
+<link rel="stylesheet" href="{% static 'commentpost/css/addcomment.css' %}" />
+<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" />
```
Comment (aloka-cuelogic) [inline commentpost/templates/commentpost/comments.html]: Please make one header page include that in all pages and put all third party references in there

File: commentpost/views.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                "postdata": PostInfo.objects.all(),
+                "userdata": User.objects.all()}
+            return HttpResponseRedirect(reverse
+                                        ('comment:addcomment',
+                                         args=(post_id,)))
+        else:
+            return HttpResponseRedirect(reverse('comment:addcomment'),
+                                        {'form': form})
+
+    @csrf_exempt
```
Comment (aloka-cuelogic) [inline commentpost/views.py]: remove csrf_exempt from here and send actual csrf_token

File: readmorepost/urls.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import include, url
```
Comment (aloka-cuelogic) [inline readmorepost/urls.py]: remove unused import

File: post/templates/post/blogpage.html [hunk 4]
```diff
@@ -80,7 +80,8 @@
                                 </div>
                                 <div class="row">
                                     <div class="col-sm-12">
-                                        {{post.post_description|safe}}
+                                        {{post.post_description|safe|truncatewords:"5"}}
```
Comment (aloka-cuelogic) [inline post/templates/post/blogpage.html]: increase the limit from 5 to some standard limit.
5 is too small. make it to 30

**Comment by aloka-cuelogic** [comment]
@sayalis-cuelogic 

Reviewed the PR.
Minor changes required.
Please do the same. re-raise the PR after resolving conflicts.

**Comment by sayalis-cuelogic** [comment]
Worked on review comments.
Created global base.html.
Used csrf_token.

**Comment by aloka-cuelogic** [comment]
@sayalis-cuelogic 

Please resolve conflicts.

sync the code with dev branch.

**Comment by sayalis-cuelogic** [comment]
Minor changes done in base template and other templates of post and comment app.

**Comment by sayalis-cuelogic** [comment]
Changes done as per review comments. 
Added global css for template.
Added home and back link.

**Comment by aloka-cuelogic** [comment]
Tested Functionality.
Working as expected.

Merging PR.

Start with New Task.

**Pull request merged**